src/ios/SimpleCameraPreview.m

```diff
@@ -65,28 +65,32 @@ - (void) enable:(CDVInvokedUrlCommand*)command {
     // Setup session
     self.sessionManager.delegate = self.cameraRenderController;
     
-    NSDictionary *setupSessionOptions;
+    NSMutableDictionary *setupSessionOptions = [NSMutableDictionary dictionary];
     if (command.arguments.count > 0) {
         NSDictionary* config = command.arguments[0];
         @try {
             if (config[@"targetSize"] != [NSNull null] && ![config[@"targetSize"] isEqual: @"null"]) {
                 NSInteger targetSize = ((NSNumber*)config[@"targetSize"]).intValue;
-                setupSessionOptions = @{ @"targetSize" : [NSNumber numberWithInteger:targetSize] };
+                [setupSessionOptions setValue:[NSNumber numberWithInteger:targetSize] forKey:@"targetSize"];
+            }
+            NSString *captureDevice = config[@"captureDevice"];
+            if (captureDevice != [NSNull null] && ![captureDevice isEqual: @""]) {
```

Comment:
if (captureDevice && ![captureDevice isEqual:[NSNull null]] && captureDevice.length > 0)

Does this work?